File: implement/implement/18406.cpp
//
//  18406.cpp
//  implement
//
//  Created by sujeong on 2024/06/20.
//
//https://www.acmicpc.net/problem/18406

#include <iostream>
#include <string>
using namespace std;

int main(){
    string s;
    cin >> s;
    
    int front = 0;
    for(int i = 0; i < s.length()/2; i++){
        front += s[i]-'0';
    }
    
    int back = 0;
    for(int i = int(s.length()/2) ; i < s.length() ; i++){
        back += s[i]-'0';
    }
    
    front == back ? cout << "LUCKY" << endl : cout << "READY" << endl;
    
    return 0;
}

File: implement/implement/18406.swift
//
//  18406.swift
//  implement
//
//  Created by sujeong on 2024/06/20.
//
//https://www.acmicpc.net/problem/18406

import Foundation

let inputs:String = readLine()!
let arr:[Int] = inputs.map { $0.wholeNumberValue! }
print(arr.prefix(arr.count/2).reduce(0) { $0 + $1 } == arr.suffix(arr.count/2).reduce(0) { $0 + $1 } ? "LUCKY" : "READY")
